task_manager/settings.py

```diff
@@ -180,3 +181,11 @@
 MESSAGE_TAGS = {
     messages.ERROR: "danger",
 }
+
+
+ROLLBAR = {
+    "access_token": "9ca9c87db9e34aa795106a03fbf8be3d",
```

Comment:
any kind of token/secret shouldn't be stored in codebase - it's a security concern

can you please move it to environment variables?